DependencyInjection/CmfCreateExtension.php

```diff
@@ -118,9 +113,6 @@ protected function loadSecurity($config, XmlFileLoader $loader, ContainerBuilder
 
     public function loadPhpcr($config, XmlFileLoader $loader, ContainerBuilder $container)
     {
-        $container->setParameter($this->getAlias() . '.backend_type_phpcr', true);
```

Comment:
indeed, this is a copy-paste mistake from other bundles that use the doctrine mappings compiler pass. we don't need this in this bundle.